silo/management/commands/get_all_json_feeds.py

```diff
@@ -31,4 +31,4 @@ def handle(self, *args, **options):
                 print(result)
                 if result[0] == "error" or result[0] == 40:
                     logger.error("Silo_ID: %s %s" % (result[2], result[1]))
-                    send_mail("Tola-Tables Auto-Pull Failed", "table_id: %s, source_id: %s, %s %s" % (silo.pk, read.pk, result[2], result[1]), "tolatables@mercycorps.org", [silo.owner.email], fail_silently=False)
\ No newline at end of file
+                    send_mail("Tola-Tables Auto-Pull Failed", "table_id: %s, source_id: %s, %s %s" % (silo.pk, read.pk, result[2], result[1]), "tolatables@mercycorps.org", [silo.owner.email], fail_silently=False)
```

Comment:
Unless you have a really good reason, you should not hardcore values like email addresses in the code. This should be an environment variable. It does not block the PR but you should fix it any time.

Reply:
That makes sense.  Would it be possible to create or use something in a Django settings file?  Is there an existing environmental variable and/or settings variable that you would like to use?

Reply:
We don't have anything like that done. It should be created.